charts/celestia-node/templates/statefulset.yaml

```diff
@@ -207,16 +207,22 @@ spec:
           {{- if .Values.node.customReadinessProbe }}
           readinessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.node.customReadinessProbe "context" $) | nindent 12 }}
           {{- else if .Values.node.readinessProbe.enabled }}
-          readinessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.node.readinessProbe "enabled") "context" $) | nindent 12 }}
+          {{- if or (eq .Values.node.nodeType "bridge") (eq .Values.node.nodeType "full") }}
+          readinessProbe:
             exec:
               command:
                 - sh
                 - '-c'
                 - >
                   AUTH_TOKEN=$(celestia bridge auth read --node.store=/home/celestia 2>/dev/null);
+                  {{- if eq .Values.node.nodeType "bridge" }}
+                  curl --max-time 0.5 -s -X POST -H "Content-type: application/json" -H "Accept: application/json" -H "Authorization: Bearer $AUTH_TOKEN" -d '{"id":1,"jsonrpc":"2.0","method":"header.SyncWait","params":[]}' "http://localhost:26658";
+                  {{- else if eq .Values.node.nodeType "full" }}
```

Comment:
`LN` has the same probe as `FN`, so we could add it here

Reply:
noted!